test/integration_test.py

```diff
@@ -148,11 +148,21 @@ def test(self):
         if config.deployment_stage != 'prod':
             try:
                 self._test_indexing()
```

Comment:
We talked about inlining this method, too. It doesn't make sense to inline the method that handles the deletions but not the one that handles the additions.